airgoons_mikeforce_customizations/config/arsenal.hpp

```diff
@@ -403,7 +403,7 @@ class vn_whitelisted_arsenal_loadouts
 		items[] = 
 		{
 			{"FirstAidKit",{-1,-1,-1,-1}},
-			{"G_Aviator",{0,0,0,0}},
+			{"G_Aviator",{-1,-1,-1,-1},
```

Comment:
No aviators?

Reply:
Those are the stock aviators, there's another pair in the mod so I figured I'd get rid of the stock ones. I'd have done the same for the bandanas but I think the stock bandanas are different colors or otherwise distinct.